Cargo.toml

```diff
@@ -122,6 +122,7 @@ pest = "2.1.3"
 pest_derive = "2.1.0"
 pin-project-lite = "0.2.12"
 pretty-hex = "0.3.0"
+priority-queue = "2.0.3"
```

Comment:
Interesting crate, might actually be a good replacement for my homegrown priority map. But do we need it for the aggregator use case? IIUC the std binary heap should be good enough?

Reply:
We still need keyed access and priority separately. Keyed for the aggregation key, priority for the flush time.